icu4c/source/test/intltest/dtfmttst.cpp

```diff
@@ -5540,6 +5541,17 @@ void DateFormatTest::TestParseRegression13744() {
     assertEquals("Error index", inDate.length(), pos.getErrorIndex());
 }
 
+void DateFormatTest::TestAdoptCalendarLeak() {
+    UErrorCode status;
```

Comment:
This needs to be initialized in order to prevent the test from randomly crashing:
`UErrorCode status = U_ZERO_ERROR;`